.prettierrc

```diff
@@ -3,5 +3,14 @@
   "printWidth": 100,
   "singleQuote": true,
   "trailingComma": "none",
-  "plugins": ["prettier-plugin-package"]
+  "plugins": ["prettier-plugin-package"],
+  "overrides": [
```

Comment:
I think I'm missing some context on this addition.

Reply:
Before, `README` files were prettiered with the default `printWidth` of 80 and the Prettier 1 default of avoiding arrow parantheses. That way, this config applies to `package.json` files with the old values. But I would be happy to align Readme files with the remaining formatting.

Reply:
I see, thanks. (also got context from your last PR-reply).